src/source/Ice/IceAgent.c

```diff
@@ -256,12 +256,15 @@ STATUS iceAgentAddRemoteCandidate(PIceAgent pIceAgent, PCHAR pIceCandidateString
     PCHAR curr, tail, next;
     UINT32 tokenLen, portValue, remoteCandidateCount, i;
     BOOL foundIpAndPort = FALSE, freeIceCandidateIfFail = TRUE;
+    BOOL foundIp6 = FALSE;
+    CHAR ip6Buf[KVS_MAX_IPV6_ADDRESS_STRING_LEN+1];
     KvsIpAddress candidateIpAddr;
 
     CHK(pIceAgent != NULL && pIceCandidateString != NULL, STATUS_NULL_ARG);
     CHK(!IS_EMPTY_STRING(pIceCandidateString), STATUS_INVALID_ARG);
 
     MEMSET(&candidateIpAddr, 0x00, SIZEOF(KvsIpAddress));
+    MEMSET(ip6Buf, 0x00, KVS_MAX_IPV6_ADDRESS_STRING_LEN+1);
```

Comment:
ditto. I won't add comments about the formatting.

Reply:
use SIZEOF(ip6Buf) ?

Reply:
Yep, makes sense.